.gitignore

```diff
@@ -4,6 +4,7 @@
 docs/_build
 docs/spec
 docs/stories
+tmt/_version.py
```

Comment:
Ok looks like that didn't work, though I believe it should still stay here, looking at other vcs/scm version implementation.

So I think the reason why it stopped working is the setuptools_scm 8.0 release yesterday.

Reply:
> So I think the reason why it stopped working is the setuptools_scm 8.0 release yesterday.

Oh? How did it break it?

Reply:
Well, with different behaviour. I'll investigate asap.

Reply:
Listing the tarball using `bash -c "ls dist/tmt-*.tar.gz"` helped.